src/hooks.js

```diff
@@ -1,29 +1,30 @@
 import cookie from 'cookie';
 import { v4 as uuid } from '@lukeed/uuid';
```

Comment:
Since > Node 16 specified in `engines` in the package manifest you could drop this dependency if you wanted and use the [Web Crypto API](https://nodejs.org/api/webcrypto.html) to generate the UUID. I'm not trying to polish here though I just wanted to get the app running so others could have a look. :smile:

Reply:
Thanks! This was generated automatically by sveltekit, so I'm fine either way. It's a sample app, I wouldn't bother really.